libmpv/opengl_cb.h

```diff
@@ -151,6 +151,18 @@ extern "C" {
  * up until mpv_opengl_cb_uninit_gl() is called. If the name is not anything
  * you know/expected, return NULL from the function.
  *
+ * * Windowing system scaling
+ * ------------------------------------
+ *
+ * When using GL, sometimes GL rendering window is upscaled to display buffer.
+ * Typically with drm where GL framebuffer can be upscaled at later stage.
+ * In That case glMPGetNativeDisplay("opengl-cb") should return the GL window
+ * dimensions, that is 4 elements:
+ * 0 : left coordinates of window (usually 0)
+ * 1 : top coordinates of window (usually 0)
+ * 2 : width of GL window
+ * 3 : height of GL window
+ *
```

Comment:
Better rename it to `opengl-cb-params` and make it a struct.

Reply:
Making a struct is fine, but i have a few questions about that. if we make a struct for this one, i suppose it makes sense to make also a struct for the drm one ? i suppose it should be `drm-params` then ?
Would these structs need to be expose in mpv headers when using it from third party app so that the interop native struct can be reused or should the thrid party app redefine it, and we only expose a `void *` and third party app would have to redefine it ? 
Either ways, what's the best place to define those structs ? `opengl__cb.h` ?

Reply:
Yes. Preferable you can use opaque structs: `struct foo { struct something *p; };` is allowed and compiles even if `something` is never defined anywhere else in the header (not even by a recursive include). That's an opaque struct.